spec/integration/embedded_content_spec.rb

```diff
@@ -44,6 +44,7 @@
     it "returns details of the edition and its publishing organisation in the results" do
       host_edition = create(:live_edition,
                             publishing_app: "whitehall",
+                            last_edited_at: 2.days.ago,
```

Comment:
just curious, is there a reason to do 'days.ago'? if you gave it a fixed date/time would the date shenanigans be needed below, as you could just compare the exact value?

Reply:
Oooh, possibly. I'll give that a go. It's early 😅

Reply:
Done! I'm now setting the string and then parsing it, which saves a bit of messing around.